app/src/main/java/com/cornellappdev/uplift/ui/components/Home/HomeCard.kt

```diff
@@ -0,0 +1,128 @@
+package com.cornellappdev.uplift.ui.components.Home
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.ui.Alignment.Companion.TopEnd
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.models.Gym
+import com.cornellappdev.uplift.util.montserratFamily
+
+
+@Composable
+    fun homeCard(gymexample:Gym ) {
+    var paintres = remember  {mutableStateOf(if(gymexample.isFavorite()) R.drawable.ic_star_filled else R.drawable.ic_star) }
+        Box(modifier= Modifier
+            .padding(5.dp)
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .aspectRatio(2F / 1F)){
+            Card(
+                modifier = Modifier
+            ) {
```

Comment:
Since this `modifier` is the default `Modifier`, please simply delete it as well as the parentheses for code cleanup!

Also, more importantly, it doesn't seem the corner radius on this card follows the Figma designs. Jason's Figma says this should be a corner radius of 12.dp, but yours uses the default 4.dp. To change this, please use the `shape` attribute and `RoundedCornerShape` type (with which you can specify a specific corner radius).